### DISCUSSION TITLE
Delay on warp

### DISCUSSION THREAD

**Discussion by TheBeast2155** [Zrips/CMI#2369] [Feature Requests]
**Description of feature/improvement request:**
can we get sone warp delay*?
---
### Detailed explanation of what it should do, when and in what situation: ###
the warp have a intant warp (nice ) but whant sone delay on it (ex on 3 sek to thay get warp)
### Basic explanation as to why this feature is needed and/or useful to have: ###
thay dont use it for get away only, and get safe fast

**Comment by arturek1666** [comment]
Do you mean a feature that when player types /warp he has to wait and don't move for specific time?
If it's that you can already do it:
![image](https://user-images.githubusercontent.com/1558228/65166937-891ed100-da39-11e9-8f2d-815f46b2e07d.png)
Just find the WarmUps in config :)

You can also do cooldowns :) 

Well with CMI you can allow cooldown our warmups for any command you have on server.
